README.md

```diff
@@ -1,104 +1,135 @@
+| `noop` | Example of how to add a simple callback executed upon a read event. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/noop)
+| `statesynchronizer` | Application that allows users to work with `StateSynchronizer` API via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/statesynchronizer)
+| `streamcuts` | Application examples demonstrating the use of `StreamCut`s via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/streamcuts) 
+
+> Hint: Have a look to the [terminology and concepts](http://pravega.io/docs/latest/terminology/) in Pravega.
+
+## Flink-connector Examples
+| Example Name  | Description  | Language |
+| ------------- |:-----| :-----|
+| `wordcount` | Counting the words continuously from a Pravega `Stream` to demonstrate the usage of Flink connector. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/wordcount)
+
+## Scenarios
+| Example Name   | Description  | Language |
+| ------------- |:-----| :-----|
+| `turbineheatsensor` | It emulates parallel sensors producing temperature values (writers) and parallel consumers performing real-time statistics (readers) via Pravega client. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/turbineheatsensor)
+| `turbineheatprocessor` | A Flink streaming application for processing temperature data from a Pravega stream produced by the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/iot), [Scala](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/scala/io/pravega/examples/flink/iot)
+| `anomaly-detection` | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/anomaly-detection)
+
+
+# Build Instructions
+
+## Pre-requisites
+
+* Java 8
+
+* Maven 2
+
+## Pravega Build Instructions (Optional)
+
+For release builds of Pravega, the artifacts will already be in Maven Central and you will 
+not need to run this step. Conversely, if you have downloaded a nightly build of Pravega, 
+you may need to generate the latest Pravega `jar` files and publish them to your local Maven 
+repository. 
+
+To install the Pravega libraries to your local Maven repository:
 
-Install the Pravega client libraries to your local Maven repository:
 ```
 $ git clone https://github.com/pravega/pravega.git
-$./gradlew install
+$ cd pravega
+$ ./gradlew install
```

Comment:
I'm ok if we do this, but another option is that we point to the snapshots we are publishing to jfrog.

Reply:
Yes, I added a mention to the `master`snapshots published to jFrog repository.